### PR TITLE
Contrast 23556 speedracer

### PR DESCRIPTION
**Pull request by palmergs**
This is the service side of the documentation for the ruby and python agents and half of the new docs for the webserver agent.

### CODE REVIEW

File: content/installation/service/installation/DebianPackages.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+For Ubuntu, in order to authenticate the Contrast repository signature
```
[the remark this answers is not in the document]
Comment (palmergs) [reply content/installation/service/installation/DebianPackages.md]: I forgot about those. It's the full description of how to do an installation from Shane. Will the build process automatically create a menu item for them? If so, then we should delete it. If not, then they could be useful to link to from the installation page for additional information.

[the remark this answers is not in the document]
Comment (kaitlynmortimer) [reply content/installation/service/installation/DebianPackages.md]: @palmergs If we want to be able to link to them, I'll have to create an entry for them on the page we want them to live on, and they'll be visible on the site. (E.g., If they should appear on the Service > Installation page, they'll show up below the ServiceInstall.md article.)

[the remark this answers is not in the document]
Comment (palmergs) [reply content/installation/service/installation/DebianPackages.md]: Ok. Let's pull them for the time being.  Should I update the PR?

[the remark this answers is not in the document]
Comment (kaitlynmortimer) [reply content/installation/service/installation/DebianPackages.md]: @palmergs Yes, please - let's take them out of this branch.

**Pull request merged**